<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PictureResize</class>
 <widget class="QMainWindow" name="PictureResize">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button_open">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Files</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Close">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>20</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>800x600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1500x1000 </string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3000x2000 </string>
     </property>
    </item>
   </widget>
   <widget class="QTextBrowser" name="ProcessDisplay">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>470</y>
      <width>571</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>New Resolution</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StartBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>START!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Files to convert</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="NewFilesSaveDir">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>310</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>enter directory or press change, default current dir</string>
    </property>
    <property name="whatsThis">
     <string>test</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>290</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Saving directory</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangeSaveDir">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>280</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Change...</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="FileList">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>571</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Close</sender>
   <signal>clicked()</signal>
   <receiver>PictureResize</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>600</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ProcessDisplay</receiver>
   <slot>insertPlainText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel">
     <x>57</x>
     <y>497</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
